torchok/tasks/detection.py

```diff
@@ -0,0 +1,106 @@
+from typing import Any, Dict, Union
+
+import torch
+import torch.nn as nn
+from omegaconf import DictConfig
+
+from torchok.constructor import BACKBONES, DETECTION_NECKS, HEADS, TASKS
+from torchok.constructor.config_structure import Phase
+from torchok.models.backbones import BackboneWrapper
+from torchok.tasks.base import BaseTask
+
+
+@TASKS.register_class
+class SingleStageDetectionTask(BaseTask):
+    def __init__(self, hparams: DictConfig):
+        """Init SingleStageDetectionTask.
+
+        Args:
+            hparams: Hyperparameters that set in yaml file.
+        """
+        super().__init__(hparams)
+
+        # BACKBONE
+        backbone_name = self._hparams.task.params.get('backbone_name')
+        backbones_params = self._hparams.task.params.get('backbone_params', dict())
+        self.backbone = BACKBONES.get(backbone_name)(**backbones_params)
+        self.num_scales = self._hparams.task.params.get('num_scales', len(self.backbone.out_encoder_channels))
```

Comment:
Need comment that is num_scales, and why it can be less than backbone features

Reply:
It's also bad that we can't write parameter like `self.num_scales` in documentation, need to discuss on torchok weekly ***

Reply:
Agree